CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ Consider migrating to `stream-chat-android-ui-components` or `stream-chat-androi
 ### ✅ Added
 
 ### ⚠️ Changed
+- When querying for channels, `ChannelListViewModel` now uses `OfflinePlugin` based approach if it is enabled. [#2921](https://github.com/GetStream/stream-chat-android/pull/2921)
```

Comment:
I think we don't need to mention that in the CHANGELOG given the fact it's always disabled in `release`

Reply:
I think it's okay to mention it for our sake at least 🤷